packages/pros-devices/src/smart/imu.rs

```diff
@@ -376,22 +381,12 @@ impl TryFrom<pros_sys::imu_raw_s> for InertialRaw {
     }
 }
 
-/// Represents a status code returned by the Inertial Sensor.
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub struct InertialStatus(pub u32);
-
-impl InertialStatus {
-    /// Determine if the sensor is currently calibrating.
-    pub const fn calibrating(&self) -> bool {
-        self.0 & pros_sys::E_IMU_STATUS_CALIBRATING != 0
-    }
-}
-
-impl TryFrom<pros_sys::imu_status_e_t> for InertialStatus {
```

Comment:
`bitflags` provides builtin methods to convert from raw bits to a typed flag struct. These include multiple different strategies (ie. `from_bits_retain`, `from_bits_truncate`, etc...) for dealing with unknown flags, which we don't currently take an opinion on. I thought it'd be better if people just go through those, since it was the intended purpose of them anyways.